ethereum/statetransition/src/test/java/tech/pegasys/teku/statetransition/util/BlobSidecarPoolImplTest.java

```diff
@@ -124,8 +124,12 @@ public void onNewBlock_addTrackerWithBlock() {
 
   @Test
   public void onNewBlobSidecar_addTrackerWithBlobSidecarIgnoringDuplicates() {
-    final BlobSidecarOld blobSidecar =
-        dataStructureUtil.createRandomBlobSidecarBuilderOld().slot(currentSlot).build();
+    final BlobSidecar blobSidecar =
+        dataStructureUtil
+            .createRandomBlobSidecarBuilder()
+            .signedBeaconBlockHeader(
+                dataStructureUtil.randomSignedBeaconBlockHeader(currentSlot, UInt64.ZERO))
```

Comment:
We could add a `randomSignedBeaconBlockHeader(final UInt64 slot)` that set a random validator index since we're not testing it...